packages/block-library/src/image/image.js

```diff
@@ -377,6 +377,8 @@ export default function Image( {
 	const lightboxChecked =
 		!! lightbox?.enabled || ( ! lightbox && !! lightboxSetting?.enabled );
 
+	const lightboxToggleDisabled = linkDestination !== 'none';
```

Comment:
Is it possible that the `linkDestination` could be `null` or empty string or some other falsy value in which case we'd also want the toggle enabled?

Reply:
I don't believe so — [you can see the logic here](https://github.com/WordPress/gutenberg/blob/8a9cb70272edcfdfd34f3557848949eb0fd29921/packages/block-library/src/image/edit.js#L203-L227).

```
                let linkDestination = attributes.linkDestination;
                if ( ! linkDestination ) {
                        // Use the WordPress option to determine the proper default.
                        // The constants used in Gutenberg do not match WP options so a little more complicated than ideal.
                        // TODO: fix this in a follow up PR, requires updating media-text and ui component.
                        switch (
                                window?.wp?.media?.view?.settings?.defaultProps?.link ||
                                LINK_DESTINATION_NONE
                        ) {
                                case 'file':
                                case LINK_DESTINATION_MEDIA:
                                        linkDestination = LINK_DESTINATION_MEDIA;
                                        break;
                                case 'post':
                                case LINK_DESTINATION_ATTACHMENT:
                                        linkDestination = LINK_DESTINATION_ATTACHMENT;
                                        break;
                                case LINK_DESTINATION_CUSTOM:
                                        linkDestination = LINK_DESTINATION_CUSTOM;
                                        break;
                                case LINK_DESTINATION_NONE:
                                        linkDestination = LINK_DESTINATION_NONE;
                                        break;
                        }
                }

```

Reply:
ok, makes sense! Thanks for checking it 🙂